Buildenator/Generators/PropertiesStringGenerator.cs

```diff
@@ -65,6 +66,14 @@ public string GeneratePropertiesCode()
 
         {GenerateChildBuilderMethodDefinition(typedSymbol)}");
 			}
+			
+			// Generate AddTo methods with Func<ChildBuilder, ChildBuilder> for collections with child builders
+			foreach (var typedSymbol in properties.Where(IsCollectionWithChildBuilder).Where(IsNotYetDeclaredChildBuilderAddToMethod))
```

Comment:
Instead of checking IsCollectionWithChildBuilder, return directly nullable object from TryGetCollectionChildBuilderName and if it's null then we skip and if not we go ahead.
Then the returned variables could be passed to the GenerateChildBuilderAddToMethodDefinition, so we don't do the same logic twice.